myapp/spa/views.py

```diff
@@ -1,5 +1,11 @@
 from django.views.generic import TemplateView
 from django.contrib.auth.mixins import LoginRequiredMixin
+from django.contrib.auth import logout
+from django.shortcuts import redirect
 
 class SpaView(LoginRequiredMixin, TemplateView):
-    template_name = "spa/index.html"
\ No newline at end of file
+    template_name = "spa/index.html"
+
+def logout_view(request):
```

Comment:
There is no official logout button for it yet, but the url accounts/logout/ works and will log the user out

Reply:
from my understanding it looks like we are using this to log out instead right? - https://github.com/HKN-UCSD/hkn-portal/pull/5/files#diff-d9410f3fb42f24340239bdd766aeaac7211a7cfc9cb7b06f6bfcc624baf92b77R41

Reply:
Yeah, I think that's the same method in views.py that you pointed out in the beginning. It's just a simple method to be called in urls.py so that we keep the format consistent. We just don't have a button that redirects to accounts/logout yet

Reply:
Oh, sorry, I just saw that the original one you pointed out was spa/views.py. Yeah, I've been using api/views.py for logout. I'm not sure if Roger/Andrew are using that for the lgout button inside the spa (since we're using the spa for post-login)

Reply:
Yeah, we should probably keep it consistent. We should only really hit up one endpoint to logout. Since even from within the spa we can still href to whatever url we want.